### ISSUE TITLE
[BUG] in POST v2/subscriptions if notification attributes are missing an error is returned

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1952]
in POST v2/subscriptions if notification attributes are missing an error is returned
> A notification object contains the following subfields:
>  -   attributes: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included in the notification. It also defines the order in which attributes will appear in notifications when attrsFormat value is used (see "Notification messages" section).

#### Subscription request
```
POST http://localhost:1026/v2/subscriptions
Content-Type: application/json
Fiware-Service: test_casub_happy_path
Fiware-ServicePath: /test
{
	"subject": {
		"entities": [{
			"idPattern": ".*",
			"type": "room"
		}],
		"condition": {
			"attributes": ["temperature_0"],
			"expression": {
				"q": "temperature>40",
				"geometry": "point",
				"georel": "near",
				"coords": "40.6391"
			}
		}
	},
	"notification": {
		"callback": "http://localhost:1234",
		"throttling": 5
	},
	"expires": "2016-04-05T14:00:00.00Z"
}
```

#### Subscription response
```
http code: 400
date: Thu, 24 Mar 2016 16:05:37 GMT
connection: Keep-Alive
content-type: application/json
content-length: 75
{"error":"BadRequest","description":"no notification attributes specified"}
```

#### Expected response
```
http code: 201
...
content-length: 0
```

**Comment by fgalan** [comment]
Fixed in PR #2023. Please @iariasleon have a look to check if this can be closed.

**Comment by iariasleon** [comment]
LGTM and tested in:
```
  "version" : "1.0.0-next",
  "git_hash" : "75e7e8f31cdd5aa88e8516b90a41ba1127b6e0b4"
```

```
{"status": "active", "reference": "http://localhost:1234", "throttling": 0L, "expression": {"q": "", "geometry": "", "georel": "", "coords": ""}, "format": "JSON", "entities": [{"id": ".*", "isPattern": "true"}], "expiration": 1459864800L, "servicePath": "/test", "_id": ObjectId("570f72c119367b8c23fb7adb"), "conditions": [{"type": "ONCHANGE", "value": ["temperature"]}], "attrs": []}
```
